Automattic/jetpack
# Add Evaluation Recommendations context menu

## Proposed changes:
* Implement context menu (DropdownMenu) visual element
* Implement options to delete Jetpack Options for welcome banner and recommendations

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go through Welcome Flow and submit goals form
* You should be presented Recommendations section, but with context menu (3 dots in top right corner)
* Click on it, and click "Redo" - you should be presented goals form again, submit it
* Now, click on context menu, and click "Hide" - recommendations should be hidden now
